### PR TITLE
Adding Arabic-Libya locale (ar-ly)

### PR DESCRIPTION
**Pull request by kikoanis**

### CODE REVIEW

**Comment by kifcaliph** [comment]
you need to change the **dow** to be 1 to match the real day. in file src/locale/ar-ly.js

File: component.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "files": [
     "moment.js",
     "locale/af.js",
+    "locale/ar-ly.js",
```
Comment (ichernev) [inline component.json]: You don't need to change this file -- its autogenerated.

**Comment by ichernev** [comment]
@kikoanis @kifcaliph which is the first day of the week in Libya?

**Comment by kifcaliph** [comment]
it is supposed to be Sunday

**Comment by mj1856** [comment]
Please undo the change to component.json and rebase to a single commit.  Thanks!

**Comment by kikoanis** [comment]
Done... Sorry about the delay

**Comment by kikoanis** [comment]
First day of the week in Libya is Saturday.

**Comment by ichernev** [comment]
Merged in 6cf3f1ff129c6a81d56c75bb658b1d96e2eaa3da

**Pull request closed without merging**